File: go/1045.go
// Em revisão!!!!
package main

import (
	"fmt"
	"math"
)

func main() {
	var A float64
	var B float64
	var C float64

	fmt.Scan(&A)
	fmt.Scan(&B)
	fmt.Scan(&C)

	pont := math.Pow(B, 2) + math.Pow(C, 2)
	pont2 := math.Pow(A, 2)
	som := B + C

	if A >= som {
		fmt.Println("NAO E UM TRIANGULO")
	}

	if pont2 == pont {
		fmt.Println("TRIANGULO RETANGULO")
	} else if pont2 > pont {
		fmt.Println("TRIANGULO OBTUSANGULO")
	} else if pont2 < pont {
		fmt.Println("TRIANGULO ACUTANGULO")
	}

	if A == B && B == C {
		fmt.Println("TRIANGULO EQUILATERO")
	} else if A == B || B == C || C == A {
		fmt.Println("TRIANGULO ISOSCELES")
	}
}

File: Lua/Iniciante/1045.lua
-- Em revisão!
local A = io.read("n")
local B = io.read("n")
local C = io.read("n")

local pont = B^2 + C^2
local pont2 = A^2
local som = B+C

if A >= som then print("NAO E UM TRIANGULO") end
if pont2 == pont then print("TRIANGULO RETANGULO") end
if pont2 > pont then print("TRIANGULO OBTUSANGULO") end
if pont2 < pont then print("TRANGULO ACUTANGULO") end
if A == B and B == C then print("TRANGULO EQUILATERO") end
if A == B or B == C or C == A then print("TRANGULO ISOSCELES") end
